arduino-ide-extension/src/node/library-service-impl.ts

```diff
@@ -456,5 +456,6 @@ function toLibrary(
     summary: lib.getParagraph(),
     category: lib.getCategory(),
     types: lib.getTypesList(),
+    isArduinoMaintained: lib.getAuthor() === 'Arduino', // TODO check if .getMaintainer is more appropriate
```

Comment:
This information is redundant, as it is already present in `author`. Would you please move this check (`lib.getAuthor() === 'Arduino'`) inside the comparator function?

Reply:
It should be based on the `types` field:

https://arduino.github.io/arduino-cli/dev/rpc/commands/#library

We intentionally don't place any restrictions on specifying "Arduino" as author because this is appropriate to give attribution for forks of the official Arduino libraries.

We do have rules about the use of "Arduino" in the maintainer field:

https://arduino.github.io/arduino-lint/dev/rules/library/#maintainer-starts-with-arduino-lp027

But due to the legacy of not enforcing this over the history of the Arduino Library Manager violation of these rules are only warnings at the "permissive" compliance level used by the Library Manager indexer engine.

But the `types` field is under Arduino's complete control so we can be sure that when a library has the value `Arduino`, it is an official library (unfortunately this has not been applied consistently, so some official libraries also don't have this categorization, but that is something to fix in the registry, not in the IDE.